### PR TITLE
Blog app - setup and controllers

### PR DESCRIPTION
**Pull request by VTY1999**
## Set up the Blog App
- Created a new app named Blogapp
- setup controllers for the app
- Added code to handle only the 4 required URL
- Added a view file for each route with a placeholder text written in plain HTML
- Fix Linter Errors

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```
Comment (Lameck1) [inline config/database.yml]: - [ ] Looking at `config/database.yml`, we can see that we've not met the following project requirement:
  
  > - Make sure that your project has Postgres database set up.
  
  The project is currently set up with `sqlite3`. Please set it up with Postgres. This [GUIDE](https://www.geeksforgeeks.org/how-to-implement-postgresql-database-in-rails-application/#:~:text=So%20to%20add%20PostgreSQL%20database,~%3E%201.4'%20from%20gemfile.&text=After%20adding%20pg%20gem%20in%20gemfile%20run%20given%20command%20on%20terminal.&text=This%20command%20installs%20all%20necessary%20files%20for%20implement%20the%20PostgreSQL%20database.) might help.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @VTY1999, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+gem 'rubocop', '>= 1.0', '< 2.0'
+
+gem 'pg'
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem 'sprockets-rails'
+
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
+
```
Comment (andersonlebon) [inline Gemfile:18]: ## [OPTIONAL]

- [ ]  As you are already using Postgres as the database(`gem 'pg'`), there is no use in having the sqlites3 gem, kindly remove it to handle this error :point_down:

image

**Review by andersonlebon — APPROVED** [review]
## Hi @VTY1999 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**